wiki/Beatmap_Editor/AiMod/fr.md

```diff
@@ -0,0 +1,156 @@
+| Cette note d'attente a une longueur inférieure à 10 ms ! | Cartes osu!mania uniquement ! | Cela nécessitera que le joueur appuie et relâche la touche à un rythme très rapide, ce qui rend votre carte impossible d'obtenir un SS. | |
+| Cet objet chevauche un autre objet. | Cartes osu!mania uniquement ! | |
+| Cet objet est empilé sur un autre objet. | Cartes osu!mania uniquement ! | Les touches et/ou les notes spécifiées sont superposées. Cela rendra votre carte impossible d'obtenir un SS ! | Trouvez la clé ou maintenez la note et corrigez-la ou supprimez-la. |
+| Plus de 6 notes simultanément ne sont pas autorisées. | Cartes osu!mania uniquement ! | La plupart des claviers peuvent autoriser jusqu'à 6 pressions de touches à la fois. | Vérifiez votre beatmap pour vous assurer que chaque colonne a au plus 6 pressions de notes. |
+
+#### Design
+
+| Message | Précisions | Solution |
+| :-- | :-- | :-- |
+| L'image d'arrière-plan est plus grande que 2560x1440. | | |
+| Cette carte peut nécessiter un avertissement d'épilepsie, car elle contient des story-boards fréquemment basculés. | | |
+| Dossier manquant : | | |
+| Les dimensions {0} doivent être {1}x{1} | | |
+| Les dimensions de votre vidéo ne doivent pas dépasser 1024x768 pour le format en ratio 4:3. | | |
+| Les dimensions de votre vidéo ne doivent pas dépasser 1280x720 pour le format en ratio 16:9. | | |
+
+#### Timing
+
+| Message | Précisions | Solution |
+| :-- | :-- | :-- |
+| Toutes les sections de chronométrage ont un volume inférieur à 5%. | | |
+| Cette beatmap dure plus de 6 minutes. Envisagez de la raccourcir si ce n'est pas une carte de style marathon. | | |
+| Votre beatmap est inférieure à 45 secondes. Pensez à l'allonger. | | |
+| Le débit binaire audio est supérieur à 192 kbps. Pensez à recompresser en CBR 192 kbps ou VBR ~ 1.0. | | |
+| Le temps de Kiai est activé pendant moins de 15 secondes. | | |
+| Le débit binaire audio est inférieur à 128 kbps. Pensez à trouver une source de meilleure qualité. | | |
+| Le mp3 que vous utilisez est beaucoup plus long que la partie mappée. Pensez à le recadrer pour réduire la taille du fichier. | | |
+| Kiai a besoin d'un point de fin. | | |
+| Un point d'aperçu pour cette carte n'est pas défini. Pensez à en définir un dans le menu Timing. | | |
+| Deux points de synchronisation existent en même temps ! | | |
+| {0} est hors de {1}, les sections de chronométrage ont un volume inférieur à 5%. | | |
+| Plus du tiers de la carte correspond à l'heure de Kiai. Pensez à réduire cela. | | |
+| Plus de la moitié de la carte de taille TV correspond à l'heure de Kiai. Pensez à réduire cela. | | |
+| Kiai n'est pas cassé ! | | |
+| La fin de Kiai n'est pas cassée ! | | |
+| Le temps de pause n'est pas suggéré pour les cartes mania | | |
+| Le diff Easy/Normal contient trop de changements de vitesse. | | |
+| Kiai est basculé très fréquemment ! | | |
+| Plus du tiers de la carte correspond à l'heure de Kiai. Pensez à réduire cela. | | |
```

Comment:
Pourquoi cette phrase est notée 4 fois ?
```suggestion
| Plus du tiers de la map correspond au Kiai time. Pensez à réduire cela. | | |
```

Reply:
Aucune idée, elle est comme ça dans le fichier en.md d'origine